frontend/test/metabase-visual/notebook/notebook.cy.spec.js

```diff
@@ -13,8 +13,12 @@ describe("visual tests > notebook > major UI elements", () => {
   it("renders correctly", () => {
     cy.visit("/question/new");
     cy.findByText("Custom question").click();
-    cy.findByText("Sample Dataset").click();
-    cy.findByText("Orders").click();
+    cy.findByText("Sample Dataset")
+      .should("be.visible")
+      .click();
```

Comment:
is it worth adding a helper "cy.findByTextEnsureVisible" or something?

Reply:
added